api/client/trust.go

```diff
@@ -25,8 +22,8 @@ import (
 	"github.com/docker/docker/api/types"
 	registrytypes "github.com/docker/docker/api/types/registry"
 	"github.com/docker/docker/cliconfig"
-	"github.com/docker/docker/pkg/ansiescape"
```

Comment:
I believe this is the last place `ansiescape` was used, and that package can be removed now.